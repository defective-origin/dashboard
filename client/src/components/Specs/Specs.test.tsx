// ---| tests |---
import { render } from 'tests'

// ---| self |---
import Specs from './Specs.component'
// import useSpecs from './Specs.hook'
// import SpecsProvider, Specs from './Specs.context'

describe('[Specs] component', () => {
  it('should render component', () => {
    const container = render(<Specs />)

    expect(container.snapshot()).toMatchSnapshot()
  })
})

// describe('[Specs] hook', () => {
//   it('should return options', () => {
//     const { result } = renderHook(() => useSpecs())
//
//     expect(result.current).toEqual(null)
//   })
// })

// describe('[Specs] context', () => {
//   const wrapper = (props: SpecsProps) => <SpecsProvider { ...props } />
//
//   it('should return options', () => {
//     const { result } = renderHook(() => useSpecs(), { wrapper })
//
//     expect(result.current).toBeTruthy()
//   })
// })
