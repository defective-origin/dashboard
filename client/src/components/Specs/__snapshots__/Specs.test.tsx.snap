// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[Specs] component > should render component 1`] = `
<DocumentFragment>
  <div
    class="block block--y item _Specs_66ed9c"
  />
</DocumentFragment>
`;
